config/confighttp/README.md

```diff
@@ -71,6 +71,9 @@ will not be enabled.
 [cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
 [cors-cache]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
 [origin]: https://developer.mozilla.org/en-US/docs/Glossary/Origin
+[attribute-processor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md
+
+You can enable [`attribute processor`][attribute-processor] to append any http header to span's attribute using custom key. You also need to enable the "include_metadata"
```

Comment:
How is the attribute processor related to this section of `cors`?

Reply:
You mean on line 74? It doesn't, you are right. But the text itself is on a good position, IMO.

Reply:
oh... these are links :))) 

Can we move them to the bottom of the file so easier to remember/identify that these are links used in the doc?

Reply:
Good idea.

Reply:
@qingbozhang, could you do this change?

Reply:
OK, no problem.  just move line 74 or move line 70~74?  maybe 70~74 IMO.